packages/embark-ui/src/utils/utils.js

```diff
@@ -1,7 +1,10 @@
 import Convert from 'ansi-to-html';
 import qs from 'qs';
 
-import {DARK_THEME} from '../constants';
+import {DARK_THEME, BALANCE_REGEX} from '../constants';
+
+import {toWei} from 'web3-utils';
+import BigNumber from 'bignumber.js';
```

Comment:
I think `bignumber.js` package needs to be added to `packages/embark-ui`'s `package.json` in `devDependencies`.